source/agora/node/Network.d

```diff
@@ -257,4 +257,12 @@ class Network
     {
         return new RestInterfaceClient!API(address);
     }
+
+    public void sendMessage(Hash msg)
```

Comment:
This could use documentation. Such as:

```
    /***************************************************************************

        Sends the message to all the listeners.

        Params:
          msg = the message to send

    ***************************************************************************/
```

Reply:
Modified.

Reply:
@AndrejMitrovic,
Check Please~